platforms/documentation/docs/src/snippets/bestPractices/avoidComputationInConfigurationPhase-avoid/groovy/build.gradle

```diff
@@ -0,0 +1,11 @@
+// tag::avoid-this[]
+void heavyWork() {
+    println("Start heavy work")
+    Thread.sleep(50)
```

Comment:
🤔 Maybe sleep 5000 = 5 seconds so that the delay is apparent to anyone downloading and running the example.